contrib/storage-druid/src/test/java/org/apache/drill/exec/store/druid/rest/DruidQueryClientTest.java

```diff
@@ -90,23 +90,23 @@ public void executeQueryCalledNoResponsesFoundReturnsEmptyEventList()
         new ByteArrayInputStream("[]".getBytes(StandardCharsets.UTF_8.name()));
     when(httpEntity.getContent()).thenReturn(inputStream);
 
-    DruidSelectResponse response = druidQueryClient.executeQuery(QUERY);
+    DruidScanResponse response = druidQueryClient.executeQuery(QUERY);
     assertThat(response.getEvents()).isEmpty();
-    assertThat(response.getPagingIdentifiers()).isEmpty();
+    //assertThat(response.getPagingIdentifiers()).isEmpty();
```

Comment:
Please remove commented out code.

Reply:
oh... yes deffinetly... missed that during coding

Reply:
Thx!